openwisp_notifications/handlers.py

```diff
@@ -305,9 +305,8 @@ def update_superuser_notification_settings(instance, created, **kwargs):
 @receiver(
     post_save, sender=Organization, dispatch_uid='org_created_notification_setting'
 )
-def notification_setting_org_created(created, instance, **kwargs):
-    if created:
-        tasks.ns_organization_created.delay(instance.pk)
+def notification_setting_org_created(instance, **kwargs):
+    transaction.on_commit(lambda: tasks.ns_organization_created.delay(instance.pk))
```

Comment:
This logic must be executed only when an organization is created, otherwise it will be executed any time an organization is updated, which we do not want.

```suggestion
    if created:
        transaction.on_commit(lambda: tasks.ns_organization_created.delay(instance.pk))
```